workshops/personal_website/README.md

```diff
@@ -272,14 +273,13 @@ Ah, it is truly beautiful to behold.
 
 Right now we can only see our website on our own computer. Let's get a link that we can share with anyone on the internet! We'll use the previously mentioned GitHub Pages to do this.
 
-1. Open the terminal by pressing `alt + t` on the keyboard at the same time. Type in the following commands and press "Enter" after each one:
-  - `git add --all`
-  - `git commit -m "Initial commit"`
-  - `git push`
-2. GitHub will now ask for your username and password.
-  - Go ahead and enter the username and press the enter.
-  - Enter the password and press enter. _The characters won't show up on the screen, but rest assured, they are still being typed._
-3. Now try to view the website by going to `USERNAME.github.io` (make sure to replace `USERNAME` with your actual GitHub username)
+1. After going back to Cloud9, press `Alt + t` on the keyboard. This is going to open the terminal, which will allow us to run special types of commands in Cloud9. In this case, we're going to save and upload our code to GitHub. Type in the following commands:
```

Comment:
How about "In c9, press alt + t" because "after going back" might not make sense to the reader. (it makes sense to us because we watched Katie leave c9 window and go to workshop window.)